src/actions/actions.ts

```diff
@@ -2082,7 +2082,7 @@ export class UpperCaseOperator extends BaseOperator {
       await TextEditor.replace(range, text.toUpperCase());
 
       vimState.currentMode = ModeName.Normal;
-      vimState.cursorPosition = start;
+      vimState.cursorPosition = vimState.cursorPositionJustBeforeAnythingHappened[0];
```

Comment:
Why is this necessary over using start? (Also the [0] won't be multi cursor friendly like start and end are)